bin/sl-meshctl.js

```diff
@@ -144,6 +144,10 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
```

Comment:
yeah, depends on the "host" app version. I reworked this so it is a bit more generic, and can work with strong-pm or strong-scheduler (or whatever we call it).

There's kindof a fundamental problem, here, which is the client-side display code, above, is tied to the server data structure. It would be nice if whatever is returned from the server was just dumped... but json dumps look awful. I guess the server side could return both the json data... as well as some kind of template that the client could expand when it wanted to display it... 

Ouph, way too much work for now.

Anyhow, PTAL at the these, see if its better now.